internal/graphql/resolver/error.go

```diff
@@ -5,66 +5,65 @@ import (
+	ErrNoMeshSync     = errors.New(ErrNoMeshSyncCode, errors.Alert, []string{"MeshSync disabled"}, []string{}, []string{}, []string{})
+	ErrEmptyHandler   = errors.New(ErrNoMeshSyncCode, errors.Alert, []string{"Database handler not initialized"}, []string{}, []string{}, []string{})
 )
 
 func ErrCreateData(err error) error {
-	return errors.NewDefault(ErrCreateDataCode, "Error while writing meshsync data", err.Error())
+	return errors.New(ErrCreateDataCode, errors.Alert, []string{"Error while writing meshsync data", err.Error()}, []string{}, []string{}, []string{})
 }
 
 func ErrUpdateData(err error) error {
-	return errors.NewDefault(ErrCreateDataCode, "Error while updating meshsync data", err.Error())
+	return errors.New(ErrCreateDataCode, errors.Alert, []string{"Error while updating meshsync data", err.Error()}, []string{}, []string{}, []string{})
 }
 
 func ErrDeleteData(err error) error {
-	return errors.NewDefault(ErrCreateDataCode, "Error while deleting meshsync data", err.Error())
+	return errors.New(ErrCreateDataCode, errors.Alert, []string{"Error while deleting meshsync data", err.Error()}, []string{}, []string{}, []string{})
 }
 
 func ErrQuery(err error) error {
-	return errors.NewDefault(ErrQueryCode, "Error while querying data", err.Error())
+	return errors.New(ErrQueryCode, errors.Alert, []string{"Error while querying data", err.Error()}, []string{}, []string{}, []string{})
 }
 
 func ErrMeshsyncSubscription(err error) error {
-	return errors.NewDefault(ErrMeshsyncSubscriptionCode, "MeshSync Subscription failed", err.Error())
+	return errors.New(ErrMeshsyncSubscriptionCode, errors.Alert, []string{"MeshSync Subscription failed", err.Error()}, []string{}, []string{}, []string{})
 }
 
 func ErrOperatorSubscription(err error) error {
-	return errors.NewDefault(ErrOperatorSubscriptionCode, "Operator Subscription failed", err.Error())
+	return errors.New(ErrOperatorSubscriptionCode, errors.Alert, []string{"Operator Subscription failed", err.Error()}, []string{}, []string{}, []string{})
 }
 
 func ErrAddonSubscription(err error) error {
-	return errors.NewDefault(ErrAddonSubscriptionCode, "Addons Subscription failed", err.Error())
+	return errors.New(ErrAddonSubscriptionCode, errors.Alert, []string{"Addons Subscription failed", err.Error()}, []string{}, []string{}, []string{})
 }
 
 func ErrControlPlaneSubscription(err error) error {
-	return errors.NewDefault(ErrControlPlaneSubscriptionCode, "Control Plane Subscription failed", err.Error())
+	return errors.New(ErrControlPlaneSubscriptionCode, errors.Alert, []string{"Control Plane Subscription failed", err.Error()}, []string{}, []string{}, []string{})
 }
 
 func ErrSubscribeChannel(err error) error {
-	return errors.NewDefault(ErrSubscribeChannelCode, "Unable to subscribe to channel", err.Error())
+	return errors.New(ErrSubscribeChannelCode, errors.Alert, []string{"Unable to subscribe to channel", err.Error()}, []string{}, []string{}, []string{})
```

Comment:
Where do descriptions, probable cause and suggestion remediation go?

Reply:
added them now in the code